Modules/_uuidmodule.c

```diff
@@ -22,7 +22,7 @@ py_uuid_generate_time_safe(void)
 /*
  * AIX support for uuid - RFC4122
```

Comment:
Since this isn't AIX-only after all, can you edit this comment?
Also please edit the similar comment in `configure.ac` where it checks the location for `uuid.h` (don't forget to regenerate `configure` afterwards).